CONTRIBUTING.md

```diff
@@ -2,28 +2,30 @@
 
 ## Foreword ##
 
-As any piece of code or documentation, this guide can bitrot if people
-forget to update it when contributing processes, development tools, or
-the Coq ecosystem changes.  If you notice anything inaccurate or
-outdated, please signal it in a new issue, or fix it in a new pull
-request.
+As with any piece of code or documentation, this guide can bitrot if
+people forget to update it when contributing processes, development
+tools, or the Coq ecosystem changes.  If you notice anything
```

Comment:
I still think it would be better to say this in a positive way without "bitrot", but I leave the choice up to you.

Reply:
Do you have a concrete suggestion? Because I have just failed to find a better wording, that's all.

Reply:
How about "As with any documentation, this guide is most useful if it's promptly updated to reflect changes in processes, development tools, or Coq ecosystem. ..."